rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1172,6 +1174,7 @@ private void processPegoutsInBatch(
             if (releaseRequestQueue.getEntries().isEmpty()) {
```

Comment:
```suggestion
            if (pegoutEntries.isEmpty()) {
```

Just for consistency. I got a bit confused when see this instead of the variable `pegoutEntries`.

Reply:
releaseRequestQueue.getEntries() Is Used Because pegoutEntries Might Have Been Updated On Line 1144